README.md

```diff
@@ -103,7 +103,7 @@ Language | Email(ms) | URI(ms) | IP(ms) | Total(ms)
 - **C**: gcc 7.5.0 & PCRE2 10.36-2
 - **Crystal**: crystal 0.35.1 - LLVM: 8.0.0
 - **C++**: g++ 7.5.0 | Boost 1.65.1.0
-- **C#**: dotnet 5.0.201 | Mono 6.12.0.122
```

Comment:
I simplified .NET's version. `5.0.201` is the SDK version which only affects building, not running programs. Servicing releases of the .NET runtime are versioned like `5.0.1`, `5.0.2` and so on, but there's no need to mention them; the fixes are very targeted and their performance impact should be negligible, and the `apt` package does not specify an exact version either way.

Make sure to change it to `.NET 6` once the benchmarks are rerun.